pom.xml

```diff
@@ -197,9 +229,11 @@
             <optional>true</optional>
         </dependency>
         <dependency>
+            <!-- used by GraphitePickleWriter  and GraphiteWriter-->
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-pool2</artifactId>
             <version>2.3</version>
+            <optional>true</optional>
```

Comment:
This will require to bump embedded-jmxtrans to 1.1.x as it may cause backward compatibility issues. Anyway, I should already have bumped to 1.1.x when I migrated from commons-pool to commons-pool2.